### DISCUSSION TITLE
notification

### DISCUSSION THREAD

**Discussion by Rifqi-Irfansyah** [directus/directus#17291] [Archive]
how can i display the message of notification

**Comment by paescuj** [comment]
Hey @Rifqi-Irfansyah 👋 Could you add some more context? Which notification do you mean?

**Reply by Rifqi-Irfansyah** [reply]
notification from flow trigger,

**Reply by paescuj** [reply]
As you've already seen yourself, this is tracked / discussed in #13727.
